site/en/docs/privacy-sandbox/ip-protection/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'IP Protection'
+subhead: >
+  Improve user privacy by protecting their IP address from being used for tracking.
+description: >
+  A proposal to improve user privacy by protecting their IP address from being used for tracking. 
+date: 2022-12-01
+authors:
+  - anusmitaray
+---
+
+{% Aside %}
+This proposal used to previously have a different scope and was known as Gnatcatcher, or the Global Network Address Translation Combined with Audited and Trusted CDN or HTTP-Proxy Eliminating Reidentification. You can still view the [deprecated documentation](/docs/privacy-sandbox/gnatcatcher/).
```

Comment:
Maybe 'This proposal previously had a different scope ...'

Reply:
Changed this